### PR TITLE
Task/add ci

### PR DESCRIPTION
**Pull request by redfieldstefan**
Create config for Travis CI
Add Travis-CI badge in README

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +38,7 @@
     "eslint": "1.3.1",
     "eslint-config-defaults": "6.0.0",
     "eslint-plugin-filenames": "0.1.2",
-    "eslint-plugin-react": "3.3.1"
+    "eslint-plugin-react": "3.3.1",
+    "mocha": "^2.3.2"
```
Comment (david-davidson) [inline package.json:42]: In keeping with the fixed ("pinned") dependencies, let's do `2.3.2`, not `^2.3.2`--sure is a pain to remember, though. :(

**Comment by david-davidson** [comment]
Sweet! Left one comment in the diff, and let's squash those two badge commits into one:
- `git rebase -i 73ffb74^` (interactive rebase from first commit hash you're interested in, inclusive)
- in vim, hit `j` to move down a line to the second (bottom) commit, `x` four times to delete `pick`, `i` once to enter editing mode, `f` (for fixup: squash into previous commit, keep previous commit's message), `esc` (to leave editing mode), and `:wq` (plus enter) to save and quit. (If you mess up, make sure you're not in editing mode and hit `u` to undo.)
- `git log` should show only one badge commit, and `git show that_commit_hash` should show the right thing!
- `git push origin task/add-ci -f`

After that, LGTM!

**Comment by redfieldstefan** [comment]
Good notes. I have rebased the badge commits and it should be good to go. I would prefer the mode of routinely scrubbing for the ^ symbol rather than worrying about it each time.

**Comment by david-davidson** [comment]
Cool, we'll ignore it for now and take a pass over the package.json once we have all our deps in order. Merge away!

**Pull request merged**